Mathlib/Analysis/NormedSpace/Units.lean

```diff
@@ -112,19 +114,22 @@ theorem inverse_add (x : Rˣ) :
   rw [← x.val_add t ht, inverse_unit, Units.add, Units.copy_eq, mul_inv_rev, Units.val_mul,
     ← inverse_unit, Units.val_oneSub, sub_neg_eq_add]
 
-theorem inverse_one_sub_nth_order' (n : ℕ) {t : R} (ht : ‖t‖ < 1) :
+theorem inverse_oneSub_nth_order' (n : ℕ) {t : R} (ht : ‖t‖ < 1) :
```

Comment:
Some of these lemmas don't actually contain `Units.oneSub`, so I'm not sure if camelCase is actually better. Are you sure you want to rename these?

In either case:
bors d+

Reply:
fair enough, I guess that's why they are named this way in the first place. I had originally thought it was just a consequence of porting issues. Closing this.